elastio-s3-changelog/README.md

```diff
@@ -11,11 +11,18 @@ Then, the Elastio `iscan` job reads those events to perform the scanning of new
+    * [ca-central-1](https://ca-central-1.console.aws.amazon.com/cloudformation/home?region=ca-central-1#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v2/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
+    * [eu-central-1](https://eu-central-1.console.aws.amazon.com/cloudformation/home?region=eu-central-1#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v2/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
+    * [eu-west-1](https://eu-west-1.console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v2/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
+    * [eu-west-2](https://eu-west-2.console.aws.amazon.com/cloudformation/home?region=eu-west-2#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v2/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
+    * [eu-west-3](https://eu-west-3.console.aws.amazon.com/cloudformation/home?region=eu-west-3#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v2/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
+    * [us-east-1](https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v2/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
+    * [us-east-2](https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v2/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
+    * [us-west-1](https://us-west-1.console.aws.amazon.com/cloudformation/home?region=us-west-1#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v2/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
+    * [us-west-2](https://us-west-2.console.aws.amazon.com/cloudformation/home?region=us-west-2#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v2/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
 
     > **Important!** You can change the stack name, but it **MUST** start with `elastio-`. Otherwise, Elastio won't be able to access the created resources.
```

Comment:
Let's move this note before the links. I'm sure 99% of people will just ignore this note because:
- A. It appears at the bottom of the huge list of links
- B. It is displayed with grey font (which contradicts its intention to draw attention).

So instead, let's make it into just a regular paragraph before the links.